### PR TITLE
Update doc for v1.8

### PR DESCRIPTION
**Pull request by reasonerjt**
Signed-off-by: Daniel Jiang <jiangd@vmware.com>

- [X] [Accepted the DCO](https://velero.io/docs/v1.5/code-standards/#dco-sign-off). Commits without the DCO will delay acceptance.
- [X] [Created a changelog file](https://velero.io/docs/v1.5/code-standards/#adding-a-changelog) or added `/kind changelog-not-required` as a comment on this pull request.
- [ ] Updated the corresponding documentation in `site/content/docs/main`.

### CODE REVIEW

**Review by zubron — CHANGES_REQUESTED** [review]
Looks good, but the "Upgrade to 1.8" page is missing. There is a TOC entry on the side but the link is broken (see https://deploy-preview-4517--velero.netlify.app/docs/v1.8/). I think it should be enough to rename [this file](https://github.com/vmware-tanzu/velero/blob/main/site/content/docs/main/upgrade-to-1.7.md) and update the contents to reference 1.8.

**Comment by reasonerjt** [comment]
@zubron Fixed, thanks for pointing out!

**Review by a-mccarthy — CHANGES_REQUESTED** [review]
I think this looks good, but the upgrade page is still missing i think. I'm still seeing a 404 on the preview site, https://deploy-preview-4517--velero.netlify.app/docs/v1.8/upgrade-to-1.8

**Comment by reasonerjt** [comment]
Thanks @a-mccarthy  @zubron  for some reason my env has been ignoring the upgrade-to-1.8.md

Please kindly check again.

File: site/content/docs/v1.8/upgrade-to-1.8.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+If you're not yet running at least Velero v1.6, see the following:
+
+- [Upgrading to v1.1][1]
+- [Upgrading to v1.2][2]
+- [Upgrading to v1.3][3]
+- [Upgrading to v1.4][4]
+- [Upgrading to v1.5][5]
+- [Upgrading to v1.6][6]
+- [Upgrading to v1.7][7]
+
```
Comment (a-mccarthy) [inline site/content/docs/v1.8/upgrade-to-1.8.md:19]: There is some missing text here to link folks to the new compatibility matrix in the repo. Its in the /main upgrade file, but missing here. 

```
Before upgrading, check the [Velero compatibility matrix](https://github.com/vmware-tanzu/velero#velero-compatibility-matrix) to make sure your version of Kubernetes is supported by the new version of Velero.
```

**Review by a-mccarthy — CHANGES_REQUESTED** [review]

File: site/content/docs/v1.8/upgrade-to-1.8.md (hunk 1 above)
Comment (reasonerjt) [reply site/content/docs/v1.8/upgrade-to-1.8.md:19]: Added, thanks.

**Review by a-mccarthy — APPROVED** [review]
lgtm, thanks!

**Review by zubron — APPROVED** [review]
Thanks!

**Pull request merged**